task_sdk/src/airflow/sdk/execution_time/comms.py

```diff
@@ -64,23 +70,22 @@ class StartupDetails(BaseModel):
     type: Literal["StartupDetails"] = "StartupDetails"
 
 
-class XComResponse(BaseModel):
```

Comment:
Changed the names in these files to avoid naming conflicts with classes in `airflow.sdk.api.datamodels._generated` (the auto-generated classes)

Reply:
Also instead of creating new classes, just extended them to add `type` literals

Reply:
Yeah, makes sense, I think when I first created these classes we didn't have the auto-generated responses